installer/stretch/precompile.sh

```diff
@@ -10,16 +10,21 @@ source ./lib/functions.sh
 
 function main()
 {
+    local KEEPALIVE_PID
     for extension in extensions/*
     do
         extension_name="${extension%.sh}"
         extension_name="${extension_name#extensions/}"
         echo -n "Installing ${extension}..."
+        bash -c 'for i in {0..20..1}; do sleep 30 && echo -n "."; done' &
```

Comment:
what is the last `..1` doing in `{0..20..1}` range?

Reply:
Increment. We could probably leave it off :)

Reply:
👍  never saw it before :)